robottelo/cli/contentview.py

```diff
@@ -35,14 +35,256 @@
+                ' "content-view-filter" or "content-view-filter-id".'
+            )
+        cls.command_sub = 'create'
+        result = cls.execute(
+            cls._construct_command(options), output_format='csv')
+
+        # Extract new CV filter rule ID if it was successfully created
+        if len(result) > 0 and 'id' in result[0]:
+            cvfr_id = result[0]['id']
+            # CV filter rule can only be fetched by specifying either
+            # content-view-filter-id or content-view-filter + content-view-id.
+            # Passing these options to info command
+            info_options = {
+                'content-view-id': options.get('content-view-id'),
+                'content-view-filter': options.get('content-view-filter'),
+                'content-view-filter-id': options.get(
+                    'content-view-filter-id'),
+                'id': cvfr_id,
+            }
+            result = cls.info(info_options)
+        return result
+
+    @classmethod
+    def update(cls, options=None):
+        """Update a filter rule. The parameters included should be based upon
+        the filter type.
+
+        Usage::
+
+            content-view filter rule update [OPTIONS]
+
+        Options::
+
+            --content-view CONTENT_VIEW_NAME        Content view name to search
+                                                    by
+            --content-view-filter CONTENT_VIEW_FILTER_NAME  Name to search by
+            --content-view-filter-id CONTENT_VIEW_FILTER_ID filter identifier
+            --content-view-id CONTENT_VIEW_ID       content view numeric
+                                                    identifier
+            --end-date END_DATE                     erratum: end date
+                                                    (YYYY-MM-DD)
+            --errata-id ERRATA_ID                   erratum: id
+            --id ID                                 rule identifier
+            --max-version MAX_VERSION               package: maximum version
+            --min-version MIN_VERSION               package: minimum version
+            --name NAME                             Name to search by
+            --new-name NEW_NAME                     package or package group:
+                                                    name
+            --start-date START_DATE                 erratum: start date
+                                                    (YYYY-MM-DD)
+            --types TYPES                           erratum: types
+                                                    (enhancement, bugfix,
+                                                    security)
+                                                    Comma separated list of
+                                                    values.
+            --version VERSION                       package: version
+
+        """
+        return super(ContentViewFilterRule, cls).update(options)
+
+
+class ContentViewFilter(Base):
```

Comment:
Is any reason why we not put both classes in separate module `contentviewfilter`?